docs/website/01_framework/controlRoutines.md

```diff
@@ -167,3 +173,19 @@ struct StandardArcLength {
     std::optional<double> psi;
   };
 ```
+
+## Automatic step-sizing for path-following techniques
+
+Instead of using a constant step size, the step size can be automatically adapted for efficient computations.
+The automatic step-sizing is implemented according to Ramm[@wunderlich_strategies_1981] .
+The step size can be scaled as shown below:
+
+```cpp
+if (targetIterations != 0 && subsidiaryArgs.actualStep > 0) {
```

Comment:
This is an implenetation detail can you state this in latex?

Reply:
Can you also point here to the correct header, where a user can find it?

Reply:
done